src/game.py

```diff
@@ -13,21 +14,39 @@ def __init__(self, ui=UserInterface(), message=Message()):
         self.validator = Validator()
         self.symbol = SymbolOptions()
         self.ui = ui
-        self.message = message
+        self.set_language(message)
         self.game_board = self.board.starter_board
         self.player_one = "X"
         self.player_two = "O"
         self.total_marks_on_board = 0
         self.playing = True
 
+    def set_language(self, message):
+        self.message = message
+
+    def change_language(self):
+        self.ui.display_message(self.message.choose_language())
+        user_input = self.ui.get_user_input()
+        valid_user_input = self.validator.is_valid_menu_or_language_choice(user_input)
+        while valid_user_input is False:
+            self.ui.display_message(self.message.invalid_choose_language_input())
+            user_input = self.ui.get_user_input()
+            valid_user_input = self.validator.is_valid_menu_or_language_choice(
+                user_input
+            )
+        if user_input == "2":
+            self.set_language(SpanishMessage())
+
     def change_symbols(self):
         self.ui.display_message(self.message.menu())
         user_input = self.ui.get_user_input()
-        valid_user_input = self.validator.is_valid_menu_choice(user_input)
+        valid_user_input = self.validator.is_valid_menu_or_language_choice(user_input)
```

Comment:
Naming note: having "or" or "and" in a name often indicates that a method may be doing too many jobs. My guess would be that either the method is in fact generic enough that this could remain `is_valid_menu_choice` (the language choice is technically a menu choice after all), or there should be two methods here.